### ISSUE TITLE
New request:  Encyclopedia of math 

### ISSUE THREAD

**Issue by DmitriyAvdeev** [openzim/zim-requests#964]
*Please use the following format for a ZIM creation request (and delete unnecessary information)*

- Website URL: https://encyclopediaofmath.org
- License: **Creative Commons ** 
- Desired ZIM Title: **Encyclopedia of Math**
- Desired ZIM Description: **Site encyclopediaofmath.org**
- Desired ZIM Icon –png (URL or attach one): https://kiwix.org/favicon.ico
- Language (ISO 639-3): **UTF-8**
- Is this a MediaWiki?: **yes**

**Comment by RavanJAltaie** [comment]
Recipe created
https://farm.openzim.org/recipes/encyclopediaofmath.org_en_all
I'll update the library file once ready

**Comment by Popolechien** [comment]
Duplicate of #92 (they say they have moved and updated their site, so the request was legit) but @RavanJAltaie can you please make sure that we don't have two recipes pointing at the same site?

**Comment by RavanJAltaie** [comment]
@benoit74 the recipe failed with the error below:
```
(node:13) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)
[error] [2024-05-08T08:59:50.796Z] Failed to run mwoffliner after [4s]: {
	"stack": "Error: Input buffer contains unsupported image format",
	"message": "Input buffer contains unsupported image format"
}
[error] [2024-05-08T08:59:50.796Z] 

**********

Input buffer contains unsupported image format

**********



```
I didn't but an image in the recipe, what does that mean?

**Comment by benoit74** [comment]
This looks like a scraper bug, or at least I can't tell what's wrong with your recipe configuration.

I will open a ticket on upstream.

**Comment by benoit74** [comment]
The problem is that the logo detected in MediaWiki configuration is in fact not available.

Please set a "ZIM favicon" in the recipe configuration, this will workaround the bug (no need at all to do it on all recipes, just this one where the MediaWiki instance seems to have an issue and the workaround is mostly straightforward).

**Comment by kelson42** [comment]
ZIM is impacted by two MWoffliner bugs: https://github.com/openzim/mwoffliner/issues/2448 and https://github.com/openzim/mwoffliner/issues/1371

**Comment by SuperCowProducts** [comment]
Was anyone actually able to archive the whole website before it went down? I'm experimenting with https://github.com/StrawberryMaster/wayback-machine-downloader but obviously the ZIM file would be the best option as well as the easiest.

[UPDATE]

PS, surely there have been other (successful) attempts in the past to archive this website other than the failed one (attempted when the site had already gone down) that appears [here](https://farm.openzim.org/recipes?name=encyclopediaofmath), right?

Has Zimit been keeping all the ZIM files created since day 1? If so, does anyone have access to such an archive and can it be searched for this 'missing' ZIM?

**Comment by Popolechien** [comment]
> Has Zimit been keeping all the ZIM files created since day 1

Not sure I understand the question so I would tentatively say "no"? 
In any case we have no archive of the encyclopedia stored anywhere. If the site is definitively down then we should probably close this issue.

**Comment by Popolechien** [comment]
Recipe ran a month ago and it looks like a case of too many files failing to download (i.e. https://github.com/openzim/mwoffliner/issues/2336)

**Comment by benoit74** [comment]
Nope, looks like their server stopped working correctly. I updated the recipe to reduce the scraping speed, asked for categories and relaunched the recipe
